grafana/grafana-github-actions
# feat(issue-commands): add command to assign issues to projects automatically

Fixes https://github.com/grafana/grafana-github-actions/issues/29

**Why?**
https://github.com/grafana/grafana-github-actions/issues/29

PR introduces new action to the type `label` called `addToProject` which automates assigning issues and pull requests to projects based on a label

**Why can't we assign "columns" automatically?**
New types of github projects do not have pre-defined columns or statuses. It is just another custom field, which would mean you would need to get a lot of custom information like `node id` for the status column and `node id` for the status value and only then you might be able to properly assign it. Further, the new projects are in beta and soon they will have more custom automation inside which would make it more proper to use the "inside project automation" instead of the grafana bot

**Additional changes**
- fixing linting errors

**Breaking changes**
- none

**How to battle-test it?**
**Issues**: Go to [this issue ](https://github.com/grafana/github-actions-testrepo/issues/31) and add a label `plugins`, wait a bit and it should be added to [this project](https://github.com/orgs/grafana/projects/76/views/12) automatically
**PRs**: Go to [this PR ](https://github.com/grafana/github-actions-testrepo/pull/32) and add a label `plugins`, wait a bit and it should be added to [this project](https://github.com/orgs/grafana/projects/76/views/12) automatically
